src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -409,7 +408,68 @@ public async Task<SyncTriggersPayload> GetSyncTriggersPayload()
             };
         }
 
-        internal static async Task<JObject> GetHostJsonExtensionsAsync(IOptionsMonitor<ScriptApplicationHostOptions> applicationHostOptions, ILogger logger)
+        internal static async Task<JObject> GetHostJsonExtensionsForDurableAsync(IOptionsMonitor<ScriptApplicationHostOptions> applicationHostOptions, ILogger logger)
```

Comment:
This isn't the approach we had discussed and agreed upon, where we decided to simply update the existing UpdateDurableFunctionConfig method to add these new values. That way we're not promoting this as a general pattern for other extensions to follow, since we already know it's problematic.

Reply:
Let's chat about this offline. I have concerns about the existing method.